frontend/packages/app/public/index.html

```diff
@@ -19,23 +19,17 @@
     -->
     <title>clutch</title>
 
-    <!--
-      Google Analytics
-      https://developers.google.com/analytics/devguides/collection/analyticsjs
-    -->
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=%REACT_APP_GA_TRACKING_ID%"></script>
     <script>
       if ("%NODE_ENV%" !== "development") {
-        window.ga =
-          window.ga ||
-          function() {
-            (ga.q = ga.q || []).push(arguments);
-          };
-        ga.l = +new Date();
-        ga("create", "%REACT_APP_GA_TRACKING_ID%", "auto");
-        ga("send", "pageview");
```

Comment:
it looks like you would need `gtag('event', 'page_view', { 'send_to': 'GA_MEASUREMENT_ID' });` to replace this based on the docs

Reply:
I think we get this for free by just initializing the config: https://developers.google.com/analytics/devguides/collection/upgrade/analyticsjs#measure_pageviews

Reply:
ah i guess that's just for an alternate tracker id :+1: